packages/config/code-hike.scss

```
@@ -657,4 +657,4 @@
   -webkit-text-decoration-style: dotted;
   text-decoration-style: dotted;
   color: inherit;
-}
+}  
```

Typically we keep the line ending on files
```suggestion
}

```